exetera/core/operations.py

```diff
@@ -3068,8 +3068,12 @@ def indexed_string_unique(indices, values, unique_result, unique_index, unique_i
 
 
 def isin_for_indexed_string_field(test_elements, indices, values):
-    if isinstance(test_elements, set):
-        test_elements = list(test_elements)
+    if test_elements is None or len(test_elements) == 0:
```

Comment:
I think these should be different conditions, if `test_elements` is None there should be an exception raise instead of returning a valid result.